.fossa.yml

```diff
@@ -0,0 +1,30 @@
+# Generated by FOSSA CLI (https://github.com/fossas/fossa-cli)
+# Visit https://fossa.com to learn more
+
+version: 2
+cli:
+  server: https://app.fossa.com
+  fetcher: custom
+  project: git@github.com:MatthewKhouzam/theia-trace-extension.git
```

Comment:
should it be theia-ide? instead of MatthewKhouzam? And will we all have access to check fossa status?